### PR TITLE
 issue #498 slideshare downloader added

### LINKED ISSUE #498: Slideshare Downloader
# Aim

- Download slides from slideshare website in PDF format from the URL.

# Details

- It will be written in python 3 and using 
docopt
img2pdf
requests
beautifulsoup4

### PR DESCRIPTION
**Pull request by vikashkumar2020**
# Description

This program download slides from Slideshare as pdf.

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation Update

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

File: Slideshare downloader/slideshare_downloader.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        print(f"fetching (slide{pg_no})")
+        file = f"slide{pg_no}.jpg"
+        urllib.request.urlretrieve(link, ".cache/"+file)
+        all_slides.append(".cache/"+file)
+        pg_no = pg_no+1
+    return all_slides
+
+
+def combine(all_slides):
+    output_name = input("\n\nEnter the name for pdf file of slides: ")
```
Comment (kaustubhgupta) [inline Slideshare downloader/slideshare_downloader.py:55]: Here you should say: "Enter the name for pdf file of slides (without extension):" as it creates confusion

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
The scrip works perfectly, just a small change to be made

**Review by kaustubhgupta — APPROVED** [review]

File: Slideshare downloader/slideshare_downloader.py (hunk 1 above)
Comment (vikashkumar2020) [reply Slideshare downloader/slideshare_downloader.py]: made changes review

**Comment by kaustubhgupta** [comment]
@santushtisharma10 @tamanna-verma @antrikshmisri

**Comment by antrikshmisri** [comment]
@vikashkumar2020 @kaustubhgupta LGTM 🚀

**Review by antrikshmisri — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]

**Pull request merged**